python/pyspark/context.py

```diff
@@ -1023,6 +1039,33 @@ def getConf(self):
         conf.setAll(self._conf.getAll())
         return conf
 
+    def install_packages(self, packages, install_driver=True):
```

Comment:
Who calls this method?

Reply:
user can call this method to install package at runtime. e.g. user can install package in pyspark shell.